docs/android-sdk.md

```diff
@@ -560,6 +567,7 @@ For this layout, there are no IDs available. This means that whenever the player
 
 - Improved bitrate switching
 - Add ‘Casting’ feature.
+  - Player's Activity Theme needs to have a `colorPrimary` defined without opacity.
```

Comment:
Could you also add this info to the documentation itself.
Maybe somewhere in the 'Add LiveryPlayerView' section?